tsconfig.json

```diff
@@ -2,14 +2,19 @@
   "compilerOptions": {
     /* Visit https://aka.ms/tsconfig.json to read more about this file */
     "target": "ESNext", /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
-    "module": "CommonJS", /* Specify what module code is generated. */
+    "module": "Node16", /* Specify what module code is generated. */
     "esModuleInterop": true, /* Emit additional JavaScript to ease support for importing CommonJS modules. This enables `allowSyntheticDefaultImports` for type compatibility. */
     "forceConsistentCasingInFileNames": true, /* Ensure that casing is correct in imports. */
     "allowSyntheticDefaultImports": true,
     "strict": true, /* Enable all strict type-checking options. */
     "noImplicitAny": false, /* Enable error reporting for expressions and declarations with an implied `any` type.. */
     "outDir": "./dist", /* Redirect output structure to the directory. */
-    "moduleResolution": "node", /* Specify how TypeScript looks up a file from a given module specifier. */
+    "moduleResolution": "Node16", /* Specify how TypeScript looks up a file from a given module specifier. */
+  },
+  "ts-node": {
```

Comment:
is this still needed? you dont use ts-node anymore

Reply:
this was me trying to salvage tsnode before giving up. feel free to remove

Reply:
it your PR

Reply:
to be fair it is 5am and I am in bed, I'll fix it tomorrow lohl

Reply:
👍

Reply:
Going to leave this here to later fix ts-node compatibility.
This goes hand-in-hand with nodemon.